tests/test.cpp

```diff
@@ -1097,3 +1097,39 @@ TEST_CASE("MinAreaBBWithRotatingCalipers", "[Geometry]") {
         REQUIRE(succ);
     }
 }
+
+TEST_CASE("SimpleNesting", "[Geometry]") {
```

Comment:
This test case is failing in the continuous testing environment. I'm fine to remove if if we can't make it work.